pkg/crc/cache/cache.go

```diff
@@ -85,12 +95,14 @@ func (c *Cache) CacheBinary() error {
 
 	for _, extractedFilePath := range extractedFiles {
```

Comment:
At this point, I think extractedFiles will always have a length of 1 or 0. Maybe we can avoid the loop ?

Reply:
@guillaumerose we put this so in future if we want to have more files like shasum or License ext. to be extracted.